client/my-sites/site-settings/disconnect-site/reconnect.jsx

```diff
@@ -0,0 +1,37 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+
+class Reconnect extends Component {
+	renderCardContent() {
+		const { translate } = this.props;
+
+		return (
+			<Card>
+				<h3>
+					{' '}
+					{ translate( 'Reconnection' ) }{' '}
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Reconnecting…' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/site-settings-disconnect-site-survey-separate-reconnection)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).